.github/workflows/label.yml

```
@@ -17,6 +17,28 @@ jobs:
+      id: check-config
+      run: |
+        if [ ! -f .github/labeler.yml ]; then
+          echo "⚠️ Warning: .github/labeler.yml not found. Skipping labeler."
+          echo "config_exists=false" >> $GITHUB_OUTPUT
+          exit 0
+        fi
+        echo "config_exists=true" >> $GITHUB_OUTPUT
+        
+    - name: Run labeler
+      if: steps.check-config.outputs.config_exists == 'true'
+      uses: actions/labeler@v4
```

actionlint (1.7.8)

the runner of "actions/labeler@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)